libs/application/templates/aosh/seminars/src/fields/Participants/index.tsx

```diff
@@ -0,0 +1,208 @@
+import { Controller, useFormContext } from 'react-hook-form'
+import {
+  Box,
+  Button,
+  InputFileUpload,
+  UploadFile,
+  UploadFileStatus,
+} from '@island.is/island-ui/core'
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+} from '@island.is/application/types'
+import { FC, useState } from 'react'
+import { FILE_SIZE_LIMIT } from '../../lib/constants'
+import { parse } from 'csv-parse'
+import { Participant } from '../../shared/types'
+import { participants as participantMessages } from '../../lib/messages'
+import { useLocale } from '@island.is/localization'
+import { DescriptionFormField } from '@island.is/application/ui-fields'
+
+interface IndexableObject {
+  [index: number]: Array<string>
+}
+const predefinedHeaders: IndexableObject = {
+  0: ['nafn', 'name'],
+  1: ['kennitala', 'ssn'],
+  2: ['netfang', 'email'],
+  3: ['sími', 'phone'],
+}
+
+const parseDataToParticipantList = (csvInput: string): Participant | null => {
+  const values = csvInput.split(';')
+  const hasNoEmptyValues =
+    !!values[0] && !!values[1] && !!values[2] && !!values[3]
+  if (!hasNoEmptyValues) {
+    return null
+  }
+  const nationalIdWithoutHyphen = values[1].replace('-', '')
+  return {
+    name: values[0],
+    nationalId: nationalIdWithoutHyphen,
+    email: values[2],
+    phoneNumber: values[3],
+    disabled: 'false',
+  }
+}
+
+const checkHeaders = (headers: string): boolean => {
+  const values = headers.split(';')
+  let validHeaders = true
+  values.map((value, index) => {
```

Comment:
```suggestion
  values.forEach((value, index) => {
```
Since there is no return in the function